README.md

```diff
@@ -1,53 +1,51 @@
-# Role Name
+# mssql (WIP)
 ![CI Testing](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This role installs and configures mssql on Red Hat Enterprise Linux 8.
+IMPORTANT: This role is WIP.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+* MS SQL requires a machine with at least 2000 megabytes of memory.
+* Optional: If you want to create a database on hosts, you must have an sql file containing the database configuration.
```

Comment:
Is this an optional requirement or not? Is there a scenario in which users would want to install MSSQL and prepare it for a database installation, but not install it right away?
Maybe I am missing something, but installing the same identical database on multiple hosts and once does not sound useful to me.

Reply:
> Is this an optional requirement or not? Is there a scenario in which users would want to install MSSQL and prepare it for a database installation, but not install it right away?

yes - just assume that if the user does not provide a .sql file, then no database setup is done

> Maybe I am missing something, but installing the same identical database on multiple hosts and once does not sound useful to me.

I think we have cleared this up in the discussions with the mssql guys - in the initial version of the role, if the user specifies to run the role against multiple hosts, we will configure each host identically

in the next version, when we support HADR, if the user specifies multiple hosts and that they want HADR, then we will set up a cluster on those hosts instead